modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrderTypes.testcase

```diff
@@ -220,6 +220,225 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11560. Verify when we complete checkout and there is more than 1 active order type for the channel, the order type with the highest priority (lower alphanumeric) is selected on the Select Order Type popup"
```

Comment:
I guess you put the wrong description for this test, this description is used for the **CanViewHighestPriorityOrderTypeIsSetByDefault** test. 

Verify when we complete checkout and there is more than 1 active order type for the channel, the order type is the one selected on the Select Order Type popup.